airflow/providers/elasticsearch/log/es_task_handler.py

```diff
@@ -135,15 +127,15 @@ def __init__(
         super().__init__(base_log_folder, filename_template)
         self.closed = False
 
-        self.client = elasticsearch.Elasticsearch(host, **es_kwargs)
         # in airflow.cfg, host of elasticsearch has to be http://dockerhostXxxx:9200
-        if USE_PER_RUN_LOG_ID and log_id_template is not None:
+        self.client = elasticsearch.Elasticsearch(host, **es_kwargs)
```

Comment:
this is just reordering the comment above the line it refers to (as best i can tell... though the comment seems nonsensical and should maybe ust be removed)